dynamorm/model.py

```diff
@@ -580,9 +580,11 @@ def update(self, conditions=None, update_item_kwargs=None, return_all=False, **k
             # update our local attrs to match what we updated
             partial_model = self.new_from_raw(resp['Attributes'], partial=True)
             for key, _ in six.iteritems(resp['Attributes']):
-                val = getattr(partial_model, key)
-                setattr(self, key, val)
-                self._validated_data[key] = val
+                # dynamorm_validate uses relaxed validation, silently drop unknown keys
```

Comment:
"Elsewhere in Dynamorm, models can be created without all fields (non-"strict" mode in Schematics), so we drop unknown keys here to be consistent."